client/css/reviewRating.scss

```diff
@@ -1,7 +1,9 @@
+$width_rating_bar:30%;
```

Comment:
Nice use of sass variables here! Now, just get the spacing right. 😄

Reply:
I was trying to use that for the rating bar, but I wasn't sure if I can pass the state as the variable. So I still renderer every state change to keep the bar percentage update. and I will delete this line.